_includes/sites-world-map.html

```diff
@@ -28,12 +28,24 @@
     // Note 2: forceLabelCoors non-false when label offset override required
     // to stop labels from overlapping in site clusters.
     const cylcUsingSiteData = [
-      { name: 'National Institute of Water & Atmospheric Research (NIWA)',
-        shortName: 'NIWA',
+      { name: 'National Institute of Water & Atmospheric Research, Auckland (NIWA)',
+        shortName: 'NIWA-1',
         devSite: true,
         forceLabelCoors: false,
         latLonDecimalDegrees: [-36.8731, 174.7773],
         umPartnerSite: true},
+      { name: 'National Institute of Water & Atmospheric Research, Wellington (NIWA)',
+        shortName: 'NIWA-2',
+        devSite: true,
+        forceLabelCoors: false,
+        latLonDecimalDegrees: [41.2865, 174.7762],
```

Comment:
Haven't checked it locally, but I think the lat needs to be negative? Does it look correct on the map with +41 instead of -41 (south)?

Reply:
Hah, good spotting - I only just checked the map and the new sites have not shown up!

Reply:
Done.